sdk/formrecognizer/Azure.AI.FormRecognizer/src/Azure.AI.FormRecognizer.csproj

```diff
@@ -2,10 +2,10 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Cognitive Services Form Recognizer client library</Description>
     <AssemblyTitle>Microsoft Azure.AI.FormRecognizer client library</AssemblyTitle>
-    <Version>4.1.0-beta.2</Version>
+    <Version>4.1.0</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>4.0.0</ApiCompatVersion>
-    <PackageTags>Microsoft Azure Form Recognizer</PackageTags>
+    <PackageTags>Microsoft Azure Form Recognizer Document Intelligence Cognitive Services Applied AI</PackageTags>
```

Comment:
All of these tags are processed separately?

Reply:
Yes. [Tags are space-delimited](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata).